src/fromager/sources.py

```diff
@@ -681,3 +688,47 @@ def pep517_build_sdist(
     )
     sdist_filename = hook_caller.build_sdist(ctx.sdists_builds)
     return ctx.sdists_builds / sdist_filename
+
+
+PKG_INFO_CONTENT = """\
+Metadata-Version: 1.0
+Name: {name}
+Version: {version}
+Summary: Fromage stub PKG-INFO
+"""
+
+
+def ensure_pkg_info(
+    *,
+    ctx: context.WorkContext,
+    req: Requirement,
+    version: Version,
+    sdist_root_dir: pathlib.Path,
+    build_dir: pathlib.Path | None = None,
+) -> bool:
+    """Ensure that sdist has a PKG-INFO file
+
+    Returns True if PKG-INFO was presence, False if file was missing. The
+    function also updates build_dir if package has a non-standard build
+    directory. Every sdist must have a PKG-INFO file in the first directory.
+    The additional PKG-INFO file in build_dir is requires for projects
```

Comment:
I should be "required" instead of "requires"

Reply:
Thanks, fixed!